### ISSUE TITLE
Plotting loss values

### ISSUE THREAD

**Issue by vinhqdang** [jcjohnson/torch-rnn#95]
Hello

I want to plot loss values to see how does it change over training process? Is there a (ready) function where I can get the log of loss?

Thanks

**Comment by jcjohnson** [comment]
The training and validation loss history are saved in the JSON checkpoints:

https://github.com/jcjohnson/torch-rnn/blob/master/train.lua#L221

There is not a file in the repo to plot them, but you can do it in just a couple lines of Python.
